.env.development

```diff
@@ -6,6 +6,7 @@ CSRF_TOKEN_API_PATH=/csrf/api/v1/token
 DISCOVERY_API_BASE_URL=http://localhost:18381
 PUBLISHER_BASE_URL=http://localhost:18400
 ECOMMERCE_BASE_URL=http://localhost:18130
+INSIGHTS_BASE_URL=
```

Comment:
I think we may want to put a URL here even if it's a bit wrong... not sure what happens exactly if this is undefined like this.  Same with .env.test below.

Reply:
See notes below about intentionally trying to keep this value empty.

Reply:
Huh, re: running a local branch of frontend-platform with a local app, there are some instructions on the README here: https://github.com/edx/frontend-platform for testing a checkout of frontend-platform with a local app - did those not work?  😬